rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -107,6 +107,10 @@ class NodeBase : public NodeBaseInterface
   rcl_guard_condition_t *
   get_notify_guard_condition() override;
 
+  RCLCPP_PUBLIC
+  rclcpp::GuardCondition *
```

Comment:
This is another case where I think we should use a reference rather than a pointer, since it should never be nullptr, right?

Reply:
Done in ccc78c5